### PR TITLE
Added `gm` fix conforming to `keys add` breaking changes 

### PR DESCRIPTION
**Pull request by adizere**
Closes: #2262 

## Description

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: scripts/gm/bin/lib-gm [hunk 1]
```diff
@@ -995,8 +995,8 @@ rpc_timeout = '15s'
 account_prefix = '${ACCOUNT_PREFIX}'
 key_name = 'wallet'
 store_prefix = 'ibc'
-gas_price = { price = 0.001, denom = '${DENOM}' }
-max_gas = 1000000
+gas_price = { price = 0.01, denom = '${DENOM}' }
+max_gas = 10000000
```
Comment (adizere) [inline scripts/gm/bin/lib-gm:999]: Note to reviewers: this change is unrelated to bug #2262.

It's just a parameter change so Hermes can submit bigger transactions by default (allows bigger gas usage).

**Review by romac — APPROVED** [review]

**Comment by ljoss17** [comment]
There is also the script `scripts/init-hermes` which uses the old key-name flag for `hermes keys add` which uses the old flag `-n` instead of the new `-k`. Should the change be made in this PR as well ?

**Comment by adizere** [comment]
> Should the change be made in this PR as well ?

Please feel free to do it, thank you!

File: scripts/init-hermes [hunk 2]
```diff
@@ -64,13 +64,13 @@ cargo build -q --locked
 # add the key seeds to the keyring of each chain
 echo "Importing keys..."
 cargo run -q --bin hermes -- -c "$CONFIG_FILE" keys add "$CHAIN_0_ID" -f "$GAIA_DATA/$CHAIN_0_ID/user_seed.json"
-cargo run -q --bin hermes -- -c "$CONFIG_FILE" keys add "$CHAIN_0_ID" -f "$GAIA_DATA/$CHAIN_0_ID/user2_seed.json" -n user2
```
Comment (adizere) [inline scripts/init-hermes:67]: Great catch, thank you for fixing Luca!
